lib/streamlit/elements/image_proto.py

```diff
@@ -49,6 +49,7 @@ def image(
         clamp=False,
         channels="RGB",
         output_format="auto",
+        help=None,
```

Comment:
In the design doc, we said that we're starting with adding the `help` keyword for form inputs; should that exclude static Streamlit elements like images and charts?

Reply:
I thought that at first but I think in the final product spec we expanded that list, see [here](https://www.notion.so/streamlit/Product-Spec-Approved-2d35bb8dc0eb4385b6da85a8fa63e7a6).

<kbd><img src='https://user-images.githubusercontent.com/76131205/104244607-9f635400-5417-11eb-8e8f-1f1dad511dd9.png'></kbd>

Reply:
Ah, thanks for that update!